tasks.yaml

```diff
@@ -0,0 +1,30 @@
+includes:
+  - create: ./tasks/create.yaml
+  - setup: ./tasks/setup.yaml
+  - deploy: ./tasks/deploy.yaml
+  - test: ./tasks/test.yaml
+
+tasks:
+  - name: setup-cluster
+    actions:
+      - task: setup:k3d-test-cluster
+
+  - name: create-single-package
+    actions:
+      - task: create:single-package
+
+  - name: create-standard-package
+    actions:
+      - task: create:standard-package
+
+  - name: deploy-single-package
+    actions:
+      - task: deploy:single-package
+
```

Comment:
Would it be valuable to add create/deploy:k3d-standard-bundle here? Might not be used in CI but I definitely run those tasks locally for dev and having them in this top level would be nice.

Reply:
I actually totally missed the addition/change to `test-uds-core` - that matches what I would want to run locally too so I'm okay with leaving as is.